openedx_filters/authentication/tests/test_filters.py

```diff
@@ -0,0 +1,38 @@
+"""
+Tests for ``authentication`` subdomain filters.
+"""
+from unittest.mock import Mock, patch
+
+from openedx_filters.authentication.filters import SessionJWTCreationRequested
+from openedx_filters.tooling import OpenEdxPublicFilter
+
+
+class TestSessionJWTCreationRequested():
+    """
+    Test class to verify standard behavior of the filters located in rendering views.
+    You'll find test suites for:
+
+    - SessionJWTCreationRequested
+    """
+
+    def test_session_jwt_creation_requested(self):
+        """
+        Test SessionJWTCreationRequested filter behavior under normal conditions.
+
+        Expected behavior:
+            - The filter should return the payload and the user.
+        """
+        payload = {'key': 'value'}
```

Comment:
Please, do not mix single and double quotes throughout the file. @cesarsilvaedunext